okd/install-packages.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+#Script to install all packages to run the Keylime agent
+
+rpm-ostree cleanup -m
+rpm-ostree upgrade
+
+# Updating base packages
```

Comment:
Can you put a comment here about why these packages need to be installed via downloaded rpm vs rpm-ostree?

Reply:
Alright, will amend the PR with this change.